Error compiling library on Raspbian

I'm getting this error when compiling `decimal-2.0.0` on Raspbian (`armv7-unknown-linux-gnueabihf` target):

```
error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/decimal-2.0.0/src/dec128.rs:229:35
    |
229 |             decQuadToString(self, buf.as_mut().as_mut_ptr());
    |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^ expected u8, found i8
    |
    = note: expected type `*mut u8`
               found type `*mut i8`
    = help: here are some functions which might fulfill your needs:
            - .offset(...)
            - .wrapping_offset(...)

error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/decimal-2.0.0/src/dec128.rs:230:39
    |
230 |             let cstr = CStr::from_ptr(buf.as_ptr());
    |                                       ^^^^^^^^^^^^ expected u8, found i8
    |
    = note: expected type `*const u8`
               found type `*const i8`
    = help: here are some functions which might fulfill your needs:
            - .offset(...)
            - .wrapping_offset(...)

error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/decimal-2.0.0/src/dec128.rs:249:38
    |
249 |             decQuadToEngString(self, buf.as_mut().as_mut_ptr());
    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^ expected u8, found i8
    |
    = note: expected type `*mut u8`
               found type `*mut i8`
    = help: here are some functions which might fulfill your needs:
            - .offset(...)
            - .wrapping_offset(...)

error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/decimal-2.0.0/src/dec128.rs:250:39
    |
250 |             let cstr = CStr::from_ptr(buf.as_ptr());
    |                                       ^^^^^^^^^^^^ expected u8, found i8
    |
    = note: expected type `*const u8`
               found type `*const i8`
    = help: here are some functions which might fulfill your needs:
            - .offset(...)
            - .wrapping_offset(...)

error: aborting due to 4 previous errors
```

It seems to happen because `c_char` on Linux ARM architecture is `u8` instead of `i8`.

--- a/src/dec128.rs
+++ b/src/dec128.rs
@@ -226,7 +226,7 @@ impl Into<u32> for d128 {
 /// payload is 0.
 impl fmt::Display for d128 {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let mut buf = [0; 43];
+        let mut buf: [c_char; 43] = [0; 43];
         unsafe {
             decQuadToString(self, buf.as_mut().as_mut_ptr());
             let cstr = CStr::from_ptr(buf.as_ptr());
@@ -246,7 +246,7 @@ impl fmt::Debug for d128 {
 /// exponential notation is used the exponent will be a multiple of 3.
 impl fmt::LowerExp for d128 {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let mut buf = [0; 43];
+        let mut buf: [c_char; 43] = [0; 43];
         unsafe {
             decQuadToEngString(self, buf.as_mut().as_mut_ptr());
             let cstr = CStr::from_ptr(buf.as_ptr());

